validator/lib/validateTileset.js

```diff
@@ -173,3 +184,103 @@ function validateTileHierarchy(root, tilesetDirectory) {
             return message;
         });
 }
+
+function checkBoundingVolume(tileBV, parentBV, tileTransform, parentTransform) {
```

Comment:
Perhaps better terminology is `innerBoundingVolume`, `outerBoundingVolume`, `innerTransform`, and `outerTransform`, just a bit more generic and matches the naming in `utility.js`.